frontend/src/metabase/visualizations/visualizations/Progress.jsx

```diff
@@ -140,13 +140,13 @@ export default class Progress extends Component {
     const column = cols[0];
     const goal = settings["progress.goal"] || 0;
 
-    const mainColor = settings["progress.color"];
-    const lightColor = Color(mainColor).lighten(0.25).rgb().string();
-    const darkColor = Color(mainColor).darken(0.3).rgb().string();
+    const colorScheme = settings["progress.color"];
```

Comment:
nit: btw why do you call it `colorScheme` it here when it's just a color? I would expect `colorScheme` to be a enum of values, but here it's just a value, right?

Reply:
@ranquild sorry, it's auto-replace, I reverted that